kotlinx-coroutines-core/common/src/flow/FlowCollector.kt

```diff
@@ -8,10 +8,25 @@ package kotlinx.coroutines.flow
  * [FlowCollector] is used as an intermediate or a terminal collector of the flow and represents
  * an entity that accepts values emitted by the [Flow].
  *
- * This interface should usually not be implemented directly, but rather used as a receiver in a [flow] builder when implementing a custom operator.
+ * This interface should usually not be implemented directly, but rather used as a receiver in a [flow] builder when implementing a custom operator
```

Comment:
```suggestion
 * This interface should usually not be implemented directly, but rather used as a receiver in a [flow] builder when implementing a custom operator,
```
Otherwise, it feels to me like it says "rather used as a receiver in a [flow] builder when (implementing a custom operator or with SAM-conversion)".